docs/source/AdministratorGuide/UserManagement/index.rst

```diff
@@ -0,0 +1,101 @@
+.. _admin_usermanagement:
```

Comment:
Can you factorize this doc with https://dirac.readthedocs.io/en/rel-v7r1/AdministratorGuide/HowTo/authentication.html ? There are quite a lot of duplicates. Maybe all the command examples should go in the how-to 

And maybe adding a few words and links to the ProxyProvider ?